lib/conjur/rack.rb

```diff
@@ -1,3 +1,26 @@
 require "conjur/rack/version"
 require "conjur/rack/authenticator"
 require "conjur/rack/path_prefix"
+require 'ipaddr'
+require 'set'
+
+module TrustedProxies
+  
+  def trusted_proxy?(ip)
+    trusted_proxies ? trusted_proxies.any? { |cidr| cidr.include?(ip) } : super
+  end
+  
+  def trusted_proxies
+    @trusted_proxies || ENV['TRUSTED_PROXIES'].try do |proxies|
+      cidrs = Set.new(proxies.split(',') + ['127.0.0.1'])
```

Comment:
Note this won't work with IP6. Maybe `trusted_proxy?` should always fall back on `super` instead.